src/services/fileStorage/model.js

```diff
@@ -1,21 +1,25 @@
 const mongoose = require('mongoose');
+const uuid = require('uuid/v4');
 
 const { Schema } = mongoose;
 
-const permissionSchema = new Schema({
-	refId: {
-		type: Schema.Types.ObjectId,
-		refPath: 'refPermModel',
-	},
-	refPermModel: {
-		type: String,
-		enum: ['user', 'role'],
+const permissionSchema = new Schema(
+	{
+		refId: {
+			type: Schema.Types.ObjectId,
```

Comment:
index? why to remove _id?

Reply:
Because this schema is only a sub-schema of `fileSchema`.